GliaWidgets/Localization.swift

```diff
@@ -0,0 +1,573 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+import Foundation
+
+// swiftlint:disable superfluous_disable_command file_length implicit_return prefer_self_in_static_references
+
+// MARK: - Strings
+
+// swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
+// swiftlint:disable nesting type_body_length type_name vertical_whitespace_opening_braces
+internal enum Localization {
+  internal enum Alert {
+    internal enum Action {
```

Comment:
Is there a way we could potentially make the indentation bigger to conform to swift standards? As this file can be generated from the command line, it's not a big issue but it would just look better

Reply:
I could take a look, probably there's a parameter.

Reply:
I investigated, apparently [there's no way of doing this](https://github.com/SwiftGen/SwiftGen/issues/390). In theory, I could modify the stencil so that it has four spaces. This would deviate us greatly from the official one. Wouldn't be the worst but we would have to decide if it's worth it to deviate so much from the original one. A benefit of being close to the original is that if they update it, we can just add our few changes easily. What do you think would be best?